ChangeLog

```diff
@@ -29,6 +29,21 @@ Version 8.3.0DEV
  - Add direct button to public page for registering when enabled.
  - Scale batch size of judge tasks dynamically.
  - Rename room to location for teams.
+ - Add option to upload contest problemset document.
+ - Improve on the advice for rejuding on config changes, only when relevant
+   advice to start a rejudging, similar for updating the scoreboard cache.
+ - For interactive problems: Add option to upload contest text/statement.
+ - Allow importing a (generic) contest warning which is displayed on the scoreboard.
+ - [UI] Use entity badges on submission pages.
```

Comment:
This is really minor and we made a dozen of other small UI changes as well. Do we want to list all of them?

Reply:
> This is really minor and we made a dozen of other small UI changes as well. Do we want to list all of them?

no, i was already against this one but we had so many that I found it was reasonable to summarize to this one. I'll remove it.